vizro-ai/docs/pages/user-guides/install.md

```diff
@@ -1,68 +1,156 @@
-# How to install Vizro-AI
+# Install Vizro-AI
 
-This guide offers instructions on installing Vizro-AI and provides comprehensive explanation on how to install, update and verify the current version of Vizro-AI.
+In this guide you'll learn how to set up the prerequisites needed for Vizro-AI, and how to install it. You'll also find out how to verify the Vizro-AI installation succeeded, find the version of Vizro-AI, and learn how to update it.
 
 ## Prerequisites
 
-- **Python**: Vizro-AI supports macOS, Linux, and Windows. It is designed to work with Python 3.8 and above. The python
-  version can be specified when setting up your virtual environment.
-- **Virtual environment**: We recommend creating a new virtual environment for each new Vizro-AI project you work on to
-  isolate the Python dependencies from those of other projects. To learn more, please see the following references about [python virtual environments](https://realpython.com/python-virtual-environments-a-primer/), [conda virtual environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
-- **OpenAI API Key**: Using Vizro-AI requires an API key, which you can get by creating an account [here](https://platform.openai.com/account/api-keys). (Users are recommended to review the third party API key section of the [disclaimer](../explanation/disclaimer.md) documentation.
-). Once you have your API key, the next step is to set your environment variable by:
-```bash
-export OPENAI_API_KEY="..."
+### Python
+Vizro-AI supports macOS, Linux, and Windows. It works with Python 3.8 and later. You can specify the version of Python to use with Vizro-AI when you set up a virtual environment.
```

Comment:
As far as I remember, Vizro-AI doesn't work with 3.8? We also say 3.9 and later in the pypi: https://pypi.org/project/vizro-ai/ but please double-check with @maxschulz-COL 

```suggestion
Vizro-AI supports macOS, Linux, and Windows. It works with Python 3.9 and later. You can specify the version of Python to use with Vizro-AI when you set up a virtual environment.
```

Reply:
Yes, let's stick with `3.9` and higher